MOPTools/MOP_Discovery/query_operations/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "ontospecies": "http://localhost:48082/blazegraph/namespace/ontomops/sparql",
+    "ontomops": "http://localhost:48082/blazegraph/namespace/ontomops/sparql"
```

Comment:
Are these two meant to be identical, i.e. the species one also pointing to the mops endpoint?

Reply:
Yes, they are meant to be the same endpoint following the refactoring of OntoMOPs TBox and ABox design.